### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by dswcodes0**
video demo: https://vimeo.com/853174429/626d935859

I added a feature where the user will not crash the app if they enter an incorrect username or platform. This makes it much more usable for users if they make any errors

### CODE REVIEW

File: statchatui/src/Compare/Compare.jsx [hunk 1]
```diff
@@ -70,12 +78,12 @@ const Compare = ({ signedInUserData }) => {
...
-    const platform = event.target.platform.value;
-    let gameName = event.target.gameName.value;
+    let platform = event.target.platform.value;
+    const gameName = event.target.gameName.value;
 
     //i did this because the api requires the apex legends xbox field to be X1 instead of Xbox, but for r6, it needs to stay xbox
     if (gameName === GAME_NAMES.APEX && platform === "Xbox") {
-      gameName = "X1";
+      platform = "X1";
     }
```
Comment (vqdo) [inline statchatui/src/Compare/Compare.jsx:87]: I think it'd be good to move this into the fetchStats function, which I assume is being used in both Home and Compare.

File: statchatui/src/Services/api.js [hunk 2]
```diff
@@ -12,6 +12,9 @@ const fetchStats = async (formData) => {
   }
   try {
     let response;
+    // if (formData.gameName === GAME_NAMES.APEX && formData.platform === "Xbox") {
+    //   formData.gameName = "X1";
+    // }
```
Comment (vqdo) [inline statchatui/src/Services/api.js:17]: Could you delete the commented out code?

**Review by vqdo — CHANGES_REQUESTED** [review]
Looks good, let me know what you think of my suggestion

File: statchatui/src/Compare/Compare.jsx (hunk 1 above)
Comment (dswcodes0) [reply statchatui/src/Compare/Compare.jsx:87]: will do

**Pull request merged**